docs/github.md

```diff
@@ -160,6 +189,8 @@ branches.
 [^2]: If you're wondering why we prefer clean commits in this project, see
 e.g.[this blog post][stacked]
 
+[auto-branch]: ./config.md#automatic-local-branch-creation
```

Comment:
I think I saw in some other PR (for GitHub pager) that it's better to not include the leading `./`. Does it work without it?

Reply:
It works without a leading slash. [View it here](https://github.com/martinvonz/jj/blob/push-woxnvonzlvmr/docs/github.md)

Reply:
Er, my comment ended up in the wrong place. I'll repeat it here:

+1 on removing `./`. The leading dot seems to break the website generated in https://github.com/martinvonz/jj/pull/2166, I made a few edits in that PR undoing such dots.

Reply:
... although, full disclosure, I don't remember if this was also a `mdbook`-specific problem or if it's a problem for `mkdocs` as well. Still, my preference would be to avoid  `./` unless we really needed it for something.